boto/vendored/regions/regions.py

```diff
@@ -0,0 +1,188 @@
+# Copyright 2014 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
The idea was to keep it as close to the source as possible to make backporting bug fixes simpler.